server/src/main/java/org/spine3/server/entity/FieldMasks.java

```diff
@@ -97,30 +96,17 @@ private FieldMasks() {
     }
 
     /**
-     * <p>Checks whether the given {@code FieldMask} is valid of not.
-     * This also includes a null check.
-     *
-     * @param fieldMask Nullable {@code FieldMask} to check.
-     * @return {@code true} if the {@code FieldMask} is valid for use, {@code} false otherwise.
-     */
-    public static boolean isValid(FieldMask fieldMask) {
-        //noinspection ConstantConditions - redundant null check.
-        return fieldMask != null && !fieldMask.getPathsList().isEmpty();
-    }
-
-    /**
-     * <p>Applies given {@code FieldMask} to a single {@link Message}.
+     * <p>Applies the given {@code FieldMask} to a single {@link Message}.
      *
      * <p>The {@code FieldMask} must be valid for this operation.
```

Comment:
This line should be now removed, as we describe the behaviour in the next sentence.